ingestion/pipelines/lineage.yaml

```diff
@@ -1,5 +1,6 @@
 serviceName: local_mysql
 filePath: test.sql
+# parseTimeout: 220 # timeout in seconds
```

Comment:
parseTimeout: 360

Reply:
@harshach have set the default timeout to 5 mins [here](https://github.com/open-metadata/OpenMetadata/blob/ad187857ada5691f61623dbce7f765806f18eca0/ingestion/src/metadata/cli/lineage.py#L39)